packages/grafana-ui/src/themes/GlobalStyles/fonts.ts

```diff
@@ -4,7 +4,9 @@ import { GrafanaTheme2 } from '@grafana/data';
 
 export function getFontStyles(theme: GrafanaTheme2) {
   const grafanaPublicPath = typeof window !== 'undefined' && window.__grafana_public_path__;
-  const fontRoot = grafanaPublicPath ? `${grafanaPublicPath}fonts/` : 'public/fonts/';
+  const relativefontRoot = grafanaPublicPath ? `${grafanaPublicPath}fonts/` : 'public/fonts/';
```

Comment:
Haven't done it before and maybe we should start. Maybe the PR description / merge commit should have dead clear instructions for what to search for to apply this patch in the future, how to validate the patch is still needed / how to validate the behavior of the patch is correct after re-applying for future updates.

Reply:
Updated description (which will be the commit message)